.gitignore

```diff
@@ -39,5 +39,4 @@
 # Ignore local seeds file
 db/seeds.local.rb
 
-# Ignore vagrant configuration/runtime files
-.vagrant/
+/config/master.key
```

Comment:
This is something I think we need to put in as a secret in kube that maps to this location, and I guess keep in our vault separately.